app/views/analyses/show.html.erb

```diff
@@ -81,7 +77,9 @@
     </div>
 
   <div class="l-analysis-advise">
-    <a href="#"><p class="text -link -medium -is-not-printable">Learn more about our methodology</p></a>
+    <%= link_to "/documents/Grow Asia Counter Methodology.pdf", target: "_blank" do %>
```

Comment:
uff, this name is not dangerous for a file. I mean, not dangerous like to kill us, but to cause some kind of trouble in the data base.